src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -65545,6 +65545,7 @@ INSERT INTO tab_groups VALUES('VOLUME_INTRACELLULAR','Intracellular','Intracellu
 INSERT INTO tab_groups VALUES('VOLUME_MUCOSA','Mucosa (total)','Mucosa (total)','ORGAN_VOLUMES',NULL,1,1,1,129,'','Organ volumes - Mucosa (total)');
 INSERT INTO tab_groups VALUES('VOLUME_PLASMA','Plasma','Plasma','ORGAN_VOLUMES',NULL,1,1,1,126,'','Organ volumes - Plasma');
 INSERT INTO tab_groups VALUES('VOLUME_VASCULAR_ENDOTHELIUM','Vascular endothelium','Vascular endothelium','ORGAN_VOLUMES',NULL,1,1,1,127,'','Organ volumes - Vascular endothelium');
+INSERT INTO tab_groups VALUES('PERMEABILITY_CALIBRATION','Intestinal permeability calibration','Parameters used to correlate experimental with in-silico intestinal permeability. Please refer to permeability.open-systems-pharmacology.org for details.','PERMEABILITY',NULL,1,6,1,152,NULL,'Intestinal permeability calibration');

```

Comment:
ok I created a shortcut...should be better

Reply:
nice!